src/renderer/shared/api/price/common/types.ts

```diff
@@ -0,0 +1,21 @@
+export type Currency = string;
+export type AssetId = string;
+
+export type PriceItem = {
+  price: number;
+  change: number;
+};
+export type AssetPrice = Record<Currency, PriceItem>;
+export type PriceObject = Record<AssetId, AssetPrice>;
+export type PriceRange = [number, string];
+export type PriceDB = {
+  assetId: AssetId;
+  currency: Currency;
+  price: number;
+  change: number;
+};
+
+export type PriceAdapter = {
```

Comment:
I assume that `Fiat` suffix can narrow `Price` meaning to our specific case
`FiatPriceAdapter`, `FiatPriceObject` etc.

Reply:
It can be not fiat, you can use crypto assets for price